charts/metering-ansible-operator/values.yaml

```diff
@@ -314,8 +314,8 @@ olm:
         version: v1
 
     annotations:
-      operatorframework.io/suggested-namespace: openshift-metering
-      operatorframework.io/cluster-monitoring: true
+      "operatorframework.io/suggested-namespace": openshift-metering
+      "operatorframework.io/cluster-monitoring": "true"
```

Comment:
@timflannagan1 I see you quoted the keys here, but not the other places.

Reply:
Fixed.